### PR TITLE
css clean up

### PR DESCRIPTION
**Pull request by Latromi**

### CODE REVIEW

**Comment by thealmightyone** [comment]
is temp.cscc currently used? I think it used to make top navbar links bold on hover. but it appears that it doesnt do that anymore.

**Comment by thayashi** [comment]
temp.scss is for cloudwan_webui, right?

**Comment by thealmightyone** [comment]
Oh wow, my bad. It is cloudwan_webui! Sorry!

File: app/css/sass/main.scss [hunk 1]
```diff
@@ -642,42 +434,9 @@ fieldset[disabled] .btn-default.focus {
   height: 40px;
 }
 
-div.form-control {
```
Comment (thayashi) [inline app/css/sass/main.scss]: This part is needed to override bootstrap form css.

File: app/css/sass/main.scss [hunk 2]
```diff
@@ -642,42 +434,9 @@ fieldset[disabled] .btn-default.focus {
...
-}
-
 input[type="checkbox"].form-control {
   box-shadow: none;
   display: inline-block;
   width: auto;
   height: auto;
 }
-
-.help-block {
```
Comment (thayashi) [inline app/css/sass/main.scss]: This part is needed to override bootstrap form css.

File: app/css/sass/main.scss [hunk 3]
```diff
@@ -642,42 +434,9 @@ fieldset[disabled] .btn-default.focus {
...
-  margin-top: 0;
-  margin-bottom: 0;
-}
-
-.tableview, .detailview, .error-message{
```
Comment (thayashi) [inline app/css/sass/main.scss]: This part is needed for error view.

File: app/css/sass/main.scss [hunk 4]
```diff
@@ -642,42 +434,9 @@ fieldset[disabled] .btn-default.focus {
...
-    top: 20px;
-    > div {
-      width: 60vw;
-      margin-left: auto;
-      margin-right: auto;
-    }
-  }
-}
-
-.bootstrap-dialog .modal-header {
```
Comment (thayashi) [inline app/css/sass/main.scss]: This part is needed to override bootstrap form css.

**Comment by stepek** [comment]
Close

**Pull request closed without merging**